operations/commit_queue.go

```diff
@@ -698,9 +697,10 @@ func (p *moduleParams) addModule(ac *legacyClient, rc *legacyClient) error {
 
 func showCQMessageForProject(ac *legacyClient, projectID string) {
 	projectRef, _ := ac.GetProjectRef(projectID)
-	err := projectRef.MergeWithParserProject("")
-	if err != nil && projectRef != nil && projectRef.CommitQueue.Message != "" {
-		grip.Info(projectRef.CommitQueue.Message)
+	if projectRef != nil && projectRef.CommitQueue.Message != "" {
+		if err := projectRef.MergeWithParserProject(""); err != nil {
```

Comment:
Sorry for not catching earlier; you can't do this for the CLI; it needs to use rest routes to communicate with Evergreen, i.e. to get the version. GetProjectRef is an example of this. I would modify that function to send over the already merged project.